### PR TITLE
Add Withdrawal Limit Feature Flag

### PR DESCRIPTION
**Pull request by phoorichet**
- [ ] I added unit tests for any code that added
- [ ] I updated the CHANGELOG.md 
- [ ] All IP is original and not copied from another source
- [ ] I assign all copyright to Loom Network for the code in the pull request

This PR adds `TGWithdrawalLimitFeature` feature flag.

### CODE REVIEW

File: cmd/loom/gateway/general_cmds.go [hunk 1]
```diff
@@ -657,6 +661,76 @@ func newSetWithdrawFeeCommand() *cobra.Command {
 	return cmd
 }
 
+func newSetMaxWithdrawLimitCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "set-max-withdrawal-limit <total-amount-limit> <per-account-amount-limit> [gateway]",
```
Comment (enlight) [inline cmd/loom/gateway/general_cmds.go]: It should be possible to change each limit individually:
```bash
set-withdrawal-limit [gateway] --total-limit <amount> --account-limit <amount>
```
So if the user specifies only `--total` then only the total limit should change.

File: jenkins.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@ set -ex
 PKG=github.com/loomnetwork/loomchain
 
 # setup temp GOPATH
+export GO111MODULE=off
```
Comment (enlight) [inline jenkins.sh]: What does this do?

Comment (phoorichet) [reply jenkins.sh]: The left over. I'll remove it.

File: cmd/loom/gateway/general_cmds.go [hunk 3]
```diff
@@ -677,37 +678,21 @@ func newSetMaxWithdrawLimitCommand() *cobra.Command {
...
 
 			var name string
-			if len(args) <= 2 || strings.EqualFold(args[2], GatewayName) {
+			if len(args) == 0 || strings.EqualFold(args[0], GatewayName) {
 				name = GatewayName
-			} else if strings.EqualFold(args[2], LoomGatewayName) {
+			} else if strings.EqualFold(args[0], LoomGatewayName) {
 				name = LoomGatewayName
 			} else {
 				return errors.New("only Gateway or LoomCoin gateway is allowed to set max withdrawal limit")
```
Comment (enlight) [inline cmd/loom/gateway/general_cmds.go]: LOOM can also be withdrawn through the Binance Gateway, so we should be able to set limits there too.

Comment (phoorichet) [reply cmd/loom/gateway/general_cmds.go]: Right. I'll add it.

File: cmd/loom/gateway/general_cmds.go [hunk 4]
```diff
@@ -657,6 +663,68 @@ func newSetWithdrawFeeCommand() *cobra.Command {
...
+			gatewayAddr, err := rpcClient.Resolve(name)
+			if err != nil {
+				return errors.Wrap(err, "failed to resolve DAppChain Gateway address")
+			}
+			gateway := client.NewContract(rpcClient, gatewayAddr.Local)
+
+			req := &tgtypes.TransferGatewaySetMaxWithdrawalLimitRequest{
+				MaxTotalDailyWithdrawalAmount:      &types.BigUInt{Value: *maxTotalAmount},
+				MaxPerAccountDailyWithdrawalAmount: &types.BigUInt{Value: *maxPerAccountAmount},
+			}
```
Comment (enlight) [inline cmd/loom/gateway/general_cmds.go]: If I only specify one of the limits on the command line then the other will get reset to zero, that shouldn't happen, only the one I specify should change. Fetch the current limits, apply the requested changes, then submit them back to the cluster.

**Pull request merged**